src/Test.FunctionalTests.BinSkim.Rules/RuleTests.cs

```diff
@@ -460,6 +498,17 @@ private HashSet<string> GetTestFilesMatchingConditions(HashSet<string> metadataC
                 result.Add(Path.Combine(testFilesDirectory, "DotnetNative_x86_VS2019_UniversalApp.exe"));
             }
 
+            if (metadataConditions.Contains(MetadataConditions.ImageIsArmBinary))
+            {
+                result.Add(Path.Combine(testFilesDirectory, "ARM_CETShadowStack_NotApplicable.exe"));
```

Comment:
>ARM_CETShadowStack_NotApplicable.exe [](http://example.com/codeflow?start=61&length=36)

nice, how did you generate this file, please add the step in md file 
\docs\FunctionalTestBuildScripts.md